common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
@jolyonb I think it looks good but can you change the class to be "wrapper-inline" since that inside element is inline. Once you make that change this should be good to merge. Thanks!